### ISSUE TITLE
NVDA does not announce the items of various edit boxes in dialogue boxes.

### ISSUE THREAD

**Issue by yogeshtaneja12** [nvaccess/nvda#11389]
<!--
I am using NVDA 2020.1 with Office 16 and values of various edit combo boxes are not getting announced.
steps to reproduce:
1. Open excel version 16.
2. Do CTRL+2 to open cell format dialogue box.
3. On number tab page navigate to "decimal point.
4. You will observe screen reader announces "decimal Point Blank"
5. Use CTRL+TAB to navigate to alignment tab / font tab.
6. Use TAB to navigate in various fields.
7.   You will observe it is not reading the values of various fields such as font, style etc.
Expected result:
the selected values should be announced while navigating to the field along with the changed values if UP/DOWNARROW is being pressed for changing them.
System configuration:
Windows 10 with Office 16.

**Comment by Adriani90** [comment]
This depends on the autor to implement a propper label. Given that we don't have a description for this including steps to reproduce and so on, I am closing this issue.
